miasm2/expression/expression.py

```diff
@@ -1116,10 +1109,9 @@ def __init__(self, *args):
         # args must be Expr
         assert all(isinstance(arg, Expr) for arg in args)
 
-        super(ExprCompose, self).__init__()
         assert isinstance(args, tuple)
         self._args = args
-        self._size = sum([arg.size for arg in args])
+        super(ExprCompose, self).__init__(sum(arg.size for arg in args))
 
     args = property(lambda self: self._args)
 
```

Comment:
I agree that you call `super __init__` for expr, but I don't see the gain to move the size `__init__` in it (erf).
What is the goal of making Expr instanciable?